developers/integrations/data-platforms/firecrawl/index.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Firecrawl
+sidebar_position: 3
+image: og/integrations/home.jpg
+---
+
+[Firecrawl](https://www.firecrawl.dev/) is an AI first web scraping tool that allows you to easily crawl and extract clean, structured data from websites. It is an API service that converts a URL into clean markdown or structured data. 
+
+## Firecrawl and Weaviate 
+Firecrawl handles the complexities of web scraping like proxies, caching, rate limits, and dynamic content, delivering markdown or JSON output that is ready to be ingested into vector databases like Weaviate.
+
+## Our Resources 
+[**Hands on Learning**](#hands-on-learning): Build your technical understanding with end-to-end tutorials.
+
+### Hands on Learning
+
+| Topic | Description | Resource | 
+| --- | --- | --- |
+| Firecrawl to Weaviate | This notebook will show you how to scrape webpages using Firecrawl and load it into Weaviate. | [Notebook](https://github.com/weaviate/recipes/blob/main/integrations/data-platforms/web-search/firecrawl/firecrawl-to-weaviate.ipynb)
```

Comment:
Consider

This notebook shows you how to scrape webpages with Firecrawl and load their contents into Weaviate.